plugin/adsk/scripts/mayaUsdTranslatorImport.mel

```diff
@@ -232,3 +268,29 @@ global proc string mayaUsdTranslatorImport_AppendFromDialog(string $currentOptio
         }
     }
 }
+
+global proc mayaUsdTranslatorImport_ViewHierCB()
+{
+    string $usdFile = currentFileDialogSelection(); 
+    string $result = `usdImportDialog $usdFile`;
+    if ("" != $result)
+    {
+        int $pc = `usdImportDialog -q -primCount`;
+        int $swc = `usdImportDialog -q -switchedVariantCount`;
+
+        text -edit -label $pc mayaUsd_NBPrimsInScopeLabel;
+        text -edit -label $swc mayaUsd_NBSwitchedVariantsLabel;
+
+        rowLayout -edit -visible true mayaUsd_NBPrimsInScopeRL;
+        rowLayout -edit -visible true mayaUsd_NBSwitchedVariantsRL;
```

Comment:
Since you have a hideInfo() helper, I think it would be a good idea to have a setInfo() helper.